### DISCUSSION TITLE
When will the typical machine learning and deep learning algorithm examples in native mojo be available?

### DISCUSSION THREAD

**Discussion by yt7589** [modular/modular#211] [General]
As mojo is an AI programming language. I want to know how to implement typical machine learning and deep learning algorithm in mojo.  At present I found that I could not find even a linear regression example in mojo examples. The most relevent one is matmul example. It will be too complicated for most developers to implement deep learning algorithms in mojo. Please give us some official implementation of typical machine learning and deep learning algorithms, such as linear regression, logistic regression, softmax regression, MLP, CNN, RNN, LSTM, GRU, Transformer......

**Comment by lattner** [comment]
+1 I'd also love to see this.  We at Modular are still getting on top of all the requests and figuring out processes, and are working hard to get the compiler downloadable etc, this is a great opportunity for community folks to experiment ahead of that

**Comment by mahmoudajawad** [comment]
Related: #136
